### ISSUE TITLE
Blue hover state in top nav and footer has insufficient contrast ratio

### ISSUE THREAD

**Issue by kydecker** [distributeaid/next-website-v2#1108]
> Update the hover state. I’m suggesting we keep the button colour but scale button size higher a bit on hover.

**Comment by ghall89** [comment]
@kydecker Could you provide more details on this issue? I'm not sure what exactly this is referring to.

**Comment by kydecker** [comment]
https://github.com/user-attachments/assets/df2987e6-6618-4b7b-94cc-5b287ae54116

I think the intent of this issue was to change the hover state for this donate button to not invert the colors of text and instead to keep the same colors but enlarge on hover.
